data/assets/events/toggle_trail.asset

```diff
@@ -0,0 +1,27 @@
+local action = asset.require('actions/toggle_trail')
+
+asset.onInitialize(function()
+  openspace.event.registerEventAction(
+    "CameraTransition",
+    action.show_trail,
+    { Before = "ApproachSphere", After = "Outside"}
```

Comment:
So, what this really means is "Going out of the approaching sphere" (i.e. Leaving an object?)? And the next one is "approaching from the outside" ()

Wondering if it would maybe be more clear to have one identifier per scenario, and remove the Before & After combination? Maybe "Approaching", "Reaching" and some good opposite words that make sense?

It may lead to more code, but I think it would make it easier to use for the general user.  Right now there are also some potential combinations that are ambiguous or could be confusing. For example, would `{ Before = "Outside", After = "Outside"}` be allowed? Or what does `{ Before = "Outside", After = "ReachedSphere"}` mean? (because the "ApproachSphere" is in between, right?).

Anyhow, l think removing the flexibility here and specify some clear options could be beneficial for the ease of use of this event type. But of course willing to discuss :)